sound/soc/sof/ipc4-topology.c

```diff
@@ -2127,6 +2154,12 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 		} else {
 			ida_free(&pipeline_ida, swidget->instance_id);
 		}
+	} else if (kcps_adjust) {
+		ret = sof_ipc4_pipeline_send_kcps(sdev, swidget, kcps_adjust);
+		if (ret < 0) {
+			dev_err(sdev->dev, "failed to increase KCPS: %d, continuing\n", ret);
```

Comment:
Then this is a `dev_warn()`

Reply:
yeah, `dev_err()` would be called by the CI, right? In fact I'm not even sure - should we make this a failure or not?

Reply:
if it is an error, then we should treat it as an error, here we are ignoring it, so it is a warning, imho